Pokedex_ios/View/RangeSlider/RangeSlider.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  RangeSlider.swift
+//  Pokedex_ios
+//
+//  Created by user on 19/7/22.
+//
+
+import Foundation
+import UIKit
+import QuartzCore
+
+class RangeSlider: UIControl {
+
+    var minimumValue = ConstantVariables.rangeValuesLowerUpper[0]
+    var maximumValue = ConstantVariables.rangeValuesLowerUpper[1]
+    var lowerValue = ConstantVariables.rangeValuesLowerUpper[0]
+    var upperValue = ConstantVariables.rangeValuesLowerUpper[1]
+
+    let trackLayer = RangeSliderTrackLayer() // = CALayer() defined in RangeSliderTrackLayer.swift
+    let lowerThumbLayer = RangeSliderThumbLayer() // CALayer()
+    let upperThumbLayer = RangeSliderThumbLayer() // CALayer()
+    let lowerThumbLabel = UILabel()
+    let upperThumbLabel = UILabel()
+    var previousLocation = CGPoint()
+
+    var trackTintColor = UIColor(white: 0.9, alpha: 1.0)
+    var trackHighlightTintColor = UIColor(named: "apply-button")
```

Comment:
This color already has a constant value for the apply-button string, please reuse that

Reply:
Now using the constant value.